build-configs/aschaffenburg/index.ts

```diff
@@ -71,20 +71,20 @@ export const iosAschaffenburgBuildConfig: iOSBuildConfigType = {
   itunesAppName: 'aschaffenburg-app',
   appleAppSiteAssociationAppIds: [`${APP_STORE_TEAM_ID}.app.aschaffenburg`],
   googleServices: {
-    clientId: '164298278764-pemn49q7v283m0tqml3q8q0ltr5h8pni.apps.googleusercontent.com',
-    reversedClientId: 'com.googleusercontent.apps.164298278764-pemn49q7v283m0tqml3q8q0ltr5h8pni',
+    clientId: '164298278764-b6gagnbo2vmickaeq2070fg4jipithsk.apps.googleusercontent.com',
+    reversedClientId: 'com.googleusercontent.apps.164298278764-b6gagnbo2vmickaeq2070fg4jipithsk',
     apiKey: 'AIzaSyBSnGYrUWfIAPcEsFya1OYvTMbjvfUyOEU',
```

Comment:
I guess this is correct, but not sure thought. Also you maybe need to add an "APNs-Auth key" (firebase -> project settings -> cloud messaging)

Reply:
Yes, in order to send messages to Apple Devices, one needs to authenticate against Apple using such a key